### DISCUSSION TITLE
Tracking helm chart versions from inside Terraform

### DISCUSSION THREAD

**Discussion by linus-ha** [dependabot/dependabot-core#8911] [Ideas]
When managing infrastructure, it is common practice to deploy helm charts via Terraform. This is done via the helm provider inside Terraform. A deployed helm chart looks something like this

```hcl
resource "helm_release" "cert_manager" {
  name             = "cert-manager"
  repository       = "https://charts.jetstack.io"
  chart            = "cert-manager"
  create_namespace = true
  namespace        = "cert-manager"
  version          = "1.12.2"

  set {
    name  = "installCRDs"
    value = "true"
  }
}
```

As you can see: The current version, chart name and repository are all set here. Therefore, looking up the newest version should be possible.  
I really would like to see this implemented!

https://registry.terraform.io/providers/hashicorp/helm/latest/docs

**Comment by netflash** [comment]
Any chance this can be looked into?

**Comment by abdulapopoola** [comment]
Thanks, could you please file a feature-request issue?

**Comment by honeyankit** [comment]
@linus-ha Could you file a feature-request issue ? Thank you.
